02 - Community Expansions/02.01 Plugins by Category/Pane Management plugins.md

```diff
@@ -15,12 +15,12 @@ Plugins to manage or cycle through panes.
 
 %% Add a bullet here and link to the plugins you'd like to categorize! %%
 
-- [[sliding-panes-obsidian|Sliding Panes (Andy's Mode)]]
-- [[pane-relief|Pane Relief]]
-- [[cycle-through-panes|Cycle through Panes]]
-- [[maximise-active-pane-obsidian|Maximise Active Pane]]
-- [[obsidian-tabs|Obsidian Tabs]]
-- [[obsidian-focus-mode|Focus Mode]]
+- [[sliding-panes-obsidian|Sliding Panes (Andy's Mode)]]: Sliding Panes! Based on the style of [Andy Matuschak's Notes](https://notes.andymatuschak.org/)
+- [[pane-relief|Pane Relief]]: Per-pane history, hotkeys for pane movement + navigation, and more
+- [[cycle-through-panes|Cycle through Panes]]: Cycle through your open Panes with `ctrl + Tab`, just like with Tabs in your Browser!, ctrl+shift+Tab for Reverse
+- [[maximise-active-pane-obsidian|Maximise Active Pane]]: Simply fills the workspace with the active pane
+- [[obsidian-tabs|Obsidian Tabs]]: Opens new leaves in tabs.
+- [[obsidian-focus-mode|Focus Mode]]: Add Focus Mode to Obsidian.
```

Comment:
This is a comment for the next line - line 23.

The comment `02 - Community Expansions/02.01 Plugins by Category/Plugins with custom views.md` below about `obsidian-fullscreen-plugin` not having acquired a comment because of the trailing space at the end of the line applies here too.

Reply:
Yup. I see what you mean. The regex that was trying to be careful about not messing with human edits was a little too strick and caused an issue with links that had a little whitespace only after them.